lib/Transforms/Scalar/ScalarReplAggregatesHLSL.cpp

```diff
@@ -3678,13 +3705,15 @@ static bool ReplaceUseOfZeroInit(Instruction *I, Value *V, DominatorTree &DT,
       continue;
 
     // Skip properly dominated users
-    if (DT.properlyDominates(BB, UI->getParent()))
+    if (DT.properlyDominates(BB, UI->getParent())) {
       continue;
+    }
```

Comment:
nit:
```suggestion
    if (DT.properlyDominates(BB, UI->getParent()))
      continue;
```
see: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements